classes/trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0, 10]
```

Comment:
```suggestion
        @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
```
This is more reliable solution

Reply:
Thank you it will be helpful when I review my code in the future.